components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -1537,6 +1542,74 @@ public void makeAPIKeysForwardCompatible(API api) throws APIManagementException
         return failedEnvironment;
     }
 
+    /**
+     * This method returns the previous versions of a given API
+     * @param api
+     * @return oldPublishedAPIList
```

Comment:
Add descriptions to this doc comments

Reply:
There is a description in this doc comment isn't it?